spec/message_configuration_server_spec.rb

```diff
@@ -283,4 +286,22 @@
       expect(subject.send(:configure_messaging_yaml)).to be_nil
     end
   end
+
+  describe "#post_activation" do
```

Comment:
Same here, should these examples expect `start` commands?

Reply:
@bdunne shouldn't this do it?

295       expect(LinuxAdmin::Service).to receive(:new).with("evmserverd").and_return(double(@spec_name, :running? => true, :restart => nil))